events-api/protobuf/v1/event.proto

```diff
@@ -23,13 +23,17 @@ enum EventCategory {
   UnknownCategory = 0;
   Pool = 1;
   Volume = 2;
+  Nexus = 3;
 }
 
 // Event Action
 enum EventAction {
   UnknownAction = 0;
   Create = 1;
   Delete = 2;
+  StateChange = 3;
+  RebuildBegin = 4;
```

Comment:
Are we simply interested in begin/end, or do we want to specify rebuild source/destination? If so, where would event specific data go into?

Reply:
We can add data specific to particular event categories.

Reply:
But why do the stats need source and destination. IMO source and destination of rebuilds are for logs, where we need to debug. But why to show that in reports?

Reply:
Stats? I thought these were generic events and stats is just one use-case ?

Reply:
hmm, for other use case it make sense. i though wrt to stats only.

Reply:
I will work on the event specific data in the future PRs.